css/css-sizing/aspect-ratio/grid-aspect-ratio-019.html

```diff
@@ -17,6 +17,9 @@
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
 
 <div id="reference-overlapped-red"></div>
-<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 100px;">
+<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 50px;">
```

Comment:
Why this change ?

Reply:
Same, to keep displaying 100x100 green rectangles like before.